recipes/jtango/build.sh

```diff
@@ -0,0 +1,4 @@
+mkdir -p ${PREFIX}/usr/share/java
```

Comment:
Omit the `usr` part.
```suggestion
mkdir -p ${PREFIX}/share/java
```

Reply:
Changed!